pkg/apis/discovery/v1alpha1/clusterducktype_types_test.go

```diff
@@ -134,9 +134,77 @@ func TestAPIVersion(t *testing.T) {
 
 	for name, tc := range tests {
 		t.Run(name, func(t *testing.T) {
-			got := APIVersion(tc.in)
+			got := tc.in.GroupVersion()
 			if !cmp.Equal(got, tc.want) {
-				t.Errorf("APIVersion (-want, +got) = %v",
+				t.Errorf("GroupVersion (-want, +got) = %v",
+					cmp.Diff(tc.want, got))
```

Comment:
Or just compare the diff to "" 2 lines above to avoid performing the same comparison twice